Makefile

```diff
@@ -0,0 +1,29 @@
+NAME    := miactl
+PACKAGE := github.com/mia-platform/$(NAME)
+GIT     := $(shell git rev-parse --short HEAD)
+SOURCE_DATE_EPOCH ?= $(shell date +%s)
+DATE    := $(shell date -u -d @${SOURCE_DATE_EPOCH} +%FT%T%Z)
+VERSION  ?= v0.0.1
+IMG_NAME := mia-platform/miactl
+IMAGE    := ${IMG_NAME}:${VERSION}
+
+default: help
+
+test:   ## Run all tests
+	@go clean --testcache && go test ./...
```

Comment:
I prefer to run test with `-race` flag set to detect data race, could we add here and in other place which run test?

Reply:
added